src/main/java/com/iexec/worker/pubsub/SubscriptionService.java

```diff
@@ -180,18 +193,19 @@ public class MessageHandler implements StompFrameHandler {
         private final String chainTaskId;
         private final String workerWalletAddress;
 
+        @NotNull
         @Override
-        public Type getPayloadType(StompHeaders headers) {
+        public Type getPayloadType(@NotNull final StompHeaders headers) {
             return TaskNotification.class;
         }
 
         @Override
-        public void handleFrame(StompHeaders headers, @Nullable Object payload) {
+        public void handleFrame(@NotNull final StompHeaders headers, @Nullable final Object payload) {
```

Comment:
Not sure we accept to use jetbrains annotation here, even if they are proposed by the IDE.

Reply:
Resolved in [09dc4c0](https://github.com/iExecBlockchainComputing/iexec-worker/pull/620/commits/09dc4c0ec20f294536aaae65d40fdcefe500c9c7)